.github/workflows/release_scripts/prisma_vulnurability_checker.py

```diff
@@ -0,0 +1,90 @@
+import requests
+import json
+import os
+import boto3
+import textwrap
+
+PRISMA_ROOT_API_URL = os.getenv("PRISMA_ROOT_API_URL")
+PRISMA_AUTHENTICATE_URL = f"{PRISMA_ROOT_API_URL}/authenticate"
+PRISMA_REGISTRY_SCAN_RESULT_URL = f"{PRISMA_ROOT_API_URL}/registry"
+PRISMA_ACCESS_KEY = os.getenv('PRISMA_ACCESS_KEY')
+PRISMA_ACCESS_KEY_SECRET = os.getenv('PRISMA_ACCESS_KEY_SECRET')
+HTTP_JSON_HEADER = 'application/json'
+# DOCKER_IMAGE_TO_CHECK is the repository name full and the tag that is used to scan it
+DOCKER_IMAGE_TO_CHECK = os.getenv("DOCKER_IMAGE_TO_CHECK")
+PRISMA_BLOCKING_VULNERABILITIES = "critical,high"
+
+
+def get_excluded_packages():
+    dynamodb_client = boto3.resource('dynamodb')
+
+    whitesource_exclusion_table = dynamodb_client.Table('prisma-excluded-packages')
+    whitesource_exclusion_entries = whitesource_exclusion_table.scan()['Items']
+
+    exclusions = set()
+    for exclusion in whitesource_exclusion_entries:
+        exclusions.add(exclusion['package'])
+
+    return exclusions
+
+
+def block_print(long_string, each_line_length=75):
```

Comment:
utility function, instead of prints long lines, this function split lines and prints them so they are easier to read.
specially useful for vulnurabilty descriptions.
example output:

```
In FasterXML jackson-databind before 2.14.0-rc1, resource
	exhaustion can occur because of a lack of a check in primitive value
	deserializers to avoid deep wrapper array nesting, when the
	UNWRAP_SINGLE_VALUE_ARRAYS feature is enabled. Additional fix version in
	2.13.4.1 and 2.12.17.1
```

instead of a single line output!